### DISCUSSION TITLE
print:h-auto

### DISCUSSION THREAD

**Discussion by damon-hall** [tailwindlabs/tailwindcss#8246] [Help]
When using the new print styles, e.g: `print:h-auto` and other size related styles they don't appear to work (are not included in the generated CSS). How can I enable them?

I should mention other (non size related) print styles work, e.g: `print:hidden` etc. but height and width plugins don't seem to be enabled for `print:`

**Comment by markquezada** [comment]
@damon-hall I am also seeing this behavior with various grid related styles. Here are some other styles that don't appear to work:

```
print:grid-cols-4
print:col-span-2
```

If I use these, nothing is generated in the output. But, if in the same file I use `print:hidden`, I do see that generated in the output.

**Comment by markquezada** [comment]
I tried this in the tailwind playground: https://play.tailwindcss.com/X6hhEpAI7D

It does seem to add the col-span-2 variant in that demo so I'm not sure why it's not working for me. I'm on v3.2.2 using the JIT compiler through elixir/phoenix's official hex package.
